azkaban-common/src/test/java/azkaban/executor/JdbcExecutorLoaderTest.java

```diff
@@ -293,6 +334,209 @@ public void testUploadExecutableNode() throws Exception {
 
   }
 
+  /* Test active executors fetch from empty executors */
+  @Test
+  public void testFetchEmptyExecutors() throws Exception {
+    if (!isTestSetup()) {
+      return;
```

Comment:
These tests are sort of integration tests and not turned on by default as their need db.